New project template does not lint on Windows: "No files matching the pattern "'src/**/*'" were found"

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/bug_report.md
-->

### Environment info

CLI version 8.11.2 on Windows

### [REQUIRED] Test case

New project template does not lint correctly.  Taken from [Stack Overflow](https://stackoverflow.com/questions/64082945):

```
i  deploying functions
Running command: npm --prefix "$RESOURCE_DIR" run lint

> functions@ lint C:\Users\fahmi\msc_mu\cloud_functions\functions
> eslint 'src/**/*'


Oops! Something went wrong! :(

ESLint: 7.9.0

No files matching the pattern "'src/**/*'" were found.
Please check for typing mistakes in the pattern.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! functions@ lint: `eslint 'src/**/*'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the functions@ lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\fahmi\AppData\Roaming\npm-cache\_logs\2020-09-26T22_01_33_578Z-debug.log

Error: functions predeploy error: Command terminated with non-zero exit code2
```

### [REQUIRED] Steps to reproduce

Just run `firebase deploy`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,2 +1,3 @@
-- Fixes issue where the ESLint config file would not be added to version control for new Typescript Cloud Functions projects (#2645)
-- Fixues issue where the CLI displayed `NaN` when choosing the default port for the Emulator UI.
+- Fixes issue where the ESLint config file would not be added to version control for new Typescript Cloud Functions projects (#2645).
+- Fixes issue where the CLI displayed `NaN` when choosing the default port for the Emulator UI.
+- Fixes the `npm run lint` (`eslint`) command in newly initalized Cloud Functions for Firebase directories on Windows (#2644).

--- a/templates/init/functions/typescript/package.lint.json
+++ b/templates/init/functions/typescript/package.lint.json
@@ -1,7 +1,7 @@
 {
   "name": "functions",
   "scripts": {
-    "lint": "eslint 'src/**/*'",
+    "lint": "eslint \"src/**/*\"",
     "build": "tsc",
     "serve": "npm run build && firebase emulators:start --only functions",
     "shell": "npm run build && firebase functions:shell",

